frontend/src/views/shop/ListShops.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+    <header-container title="My Shops">
+        <router-link
+            to="/account/shops/new"
+            type="button"
+            class="group btn btn-primary flex items-center align-middle"
+        >
+            <icon-fa6-solid:plus
+                class="-ml-1 mr-2 h-4 w-4 opacity-25 group-hover:opacity-50"
+                aria-hidden="true"
+            />
+            Add Shop
+        </router-link>
+    </header-container>
+    <main-container v-if="!shopStore.isLoading">
+        <div
+            v-if="shopStore.shops.length === 0"
+            class="text-center"
+        >
+            <svg
+                class="mx-auto h-12 w-12 text-gray-400 dark:text-neutral-500"
+                fill="none"
+                viewBox="0 0 24 24"
+                stroke="currentColor"
+                aria-hidden="true"
+            >
+                <path
+                    vector-effect="non-scaling-stroke"
+                    stroke-linecap="round"
+                    stroke-linejoin="round"
+                    stroke-width="2"
+                    d="M9 13h6m-3-3v6m-9 1V7a2 2 0 012-2h6l2 2h6a2 2 0 012 2v8a2 2 0 01-2 2H5a2 2 0 01-2-2z"
+                />
+            </svg>
+            <h3 class="mt-2 font-medium">
+                No shops
+            </h3>
+            <p class="mt-1 text-gray-500 dark:text-neutral-500">
+                Get started by adding your first Shop.
+            </p>
+            <div class="mt-6">
+                <router-link
+                    to="/account/shops/new"
+                    class="btn btn-primary group flex items-center"
+                >
+                    <icon-fa6-solid:plus
+                        class="-ml-1 mr-2 h-4 w-4 opacity-25 group-hover:opacity-50"
+                        aria-hidden="true"
+                    />
+                    Add Shop
+                </router-link>
+            </div>
+        </div>
+
+        <div
+            v-else
+            class="shadow rounded-md overflow-y-scroll md:overflow-y-hidden bg-white dark:bg-neutral-800 dark:shadow-none"
+        >
+            <data-table
+                :columns="[
+                    { key: 'favicon', name: 'Logo' },
```

Comment:
The favicon col has no title and has other styles before
`:labels="{favicon: {name: '', classOverride: true, class: 'w-11 min-w-[44px] py-3.5 px-3'}`
![image](https://github.com/FriendsOfShopware/shopmon/assets/412142/9e9d99a6-4220-4d62-90cb-b4f091796891)
![image](https://github.com/FriendsOfShopware/shopmon/assets/412142/f0162a2c-63a9-4809-99af-6c72d1f94c2b)

Reply:
fixed it with https://github.com/FriendsOfShopware/shopmon/compare/432eb466bf4543ca446cd2346cc30eb2b08145ac..5e53a5554f16e332639b661c1d125be2c9f14dfb#diff-8275cc51c7438defe929163bde969ba57fc5bc1836a132a7f579b28bdc6efca2R61